src/components/UserMapMarker.css

```diff
@@ -1,5 +1,7 @@
 .UserMapMarker-dot {
-  width: 17px;
-  height: 50px;
-  background: url(../img/monogramBlack.png) no-repeat center center;
+  width: 64px;
+  height: 64px;
+  margin-left: -32px;
```

Comment:
@lukaswelte To center the user. For the moment the center (where the user is) was the top left corner.